source/dstats/distrib.d

```diff
@@ -603,15 +603,23 @@ uint invBinomialCDF(double pVal, uint n, double p) {
     }
 }
 
+deprecated {
+    uint invBinomialCDF(double pVal, uint n, double p) {
+        import std.conv : to;
+
+        return invBinomialCDF(pVal, n.to!ulong, p).to!uint;
+    }
+}
```

Comment:
this is gonna be annoying as this overload will be taken every time anyone uses an `int` (including small enough literals) for `n`